collections/_infographics/climate/climate-change-scheme.md

```diff
@@ -0,0 +1,15 @@
+---
+layout:        infographic
+title:         "Climate change scheme"
+slug:          "climate-change-scheme"
+redirect_from: "/climate-change-scheme"
+published:     2022-07-26
+weight:        80
+tags-scopes:   [ world ]
+tags-topics:   [ climate ]
+caption:       "Climate change is not just a change of temperature – this umbrella term covers a number of interrelated phenomena. The change of one factor (e.g. higher concentrations of CO2 in the atmosphere) results in a long causal chain."
+---
+
+_Our comments on this infographic providing relevant context and useful links are [only available in Czech](https://faktaoklimatu.cz/infografiky/schema-klimaticke-zmeny) at the moment. Please check back later for the English translation._
```

Comment:
Why not using a local include with this text (with an param `url`) for all the content files?

Reply:
Good idea, I'll use an include instead.